src/main/scala/mesosphere/marathon/tasks/PortsMatcher.scala

```diff
@@ -29,7 +29,7 @@ case class PortsMatch(hostPortsWithRole: Seq[Option[PortWithRole]]) {
 /**
   * Utility class for checking if the ports resource in an offer matches the requirements of an app.
   */
-class PortsMatcher(
+class PortsMatcher private[tasks] (
```

Comment:
It makes the constructor private but the class public. If the class is private the compiler complained that it was somewhere else. It's actually not an important change.